backend/app/views/backoffice/project_developers/index.html.erb

```diff
@@ -0,0 +1,42 @@
+<table class="backoffice-table">
+  <thead>
+    <tr>
+      <th>
+        <%= t("backoffice.name") %>
+      </th>
+      <th>
+        <%= t("backoffice.account_owner") %>
+      </th>
+      <th>
+        <%= t("backoffice.account_users") %>
+      </th>
+      <th>
+        <%= t("backoffice.projects") %>
+      </th>
+      <th>
+        <%= t("backoffice.language") %>
+      </th>
+      <th>
+        <%= t("backoffice.status") %>
+      </th>
+      <th>
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @project_developers.each do |pd| %>
+      <tr>
+        <td><%= pd.name %></td>
+        <td><%= pd.account.owner.full_name %></td>
+        <td><%= pd.account.users.count + 1 %></td>
+        <td><%= pd.projects.count %></td>
```

Comment:
I guess this line and previous one will trigger db query per project developer

Reply:
Those counter caches will be good when adding sorting.